CONTRIBUTING.md

```diff
@@ -0,0 +1,106 @@
+# Contributing
+We are open to, and grateful for, any contributions made by the community.  By contributing to Redux, you agree to abide by the [code of conduct](https://github.com/rackt/redux/blob/master/CODE_OF_CONDUCT.md).
+
+## Reporting Issues and Asking Questions
+Before opening an issue, please search the [issue tracker](https://github.com/rackt/redux/issues) to make sure your issue hasn't already been reported.
+
+Please ask any general and implementation specific questions on [Stack Overflow with a Redux tag](http://stackoverflow.com/questions/tagged/redux?sort=votes&pageSize=50) for support.
+
+## Development
+
+Visit the [Issue tracker](https://github.com/rackt/redux/issues) to find a list of open issues that need attention.
+
+Fork, then clone the repo:
+```
+git clone https://github.com/your-username/redux.git
+```
+
+### Building
+
+#### Build Redux
+To build Redux:
+```
+npm run build
+```
+
+To build the lib:
```

Comment:
This is confusing. “Build the lib” presumes contributor knows what we mean by `lib`. Better to explain that the `build` task contains two subtasks: the CommonJS module-per-module build and a UMD build.